pkg/driverbuilder/builder/templates/alinux_kernel.sh

```diff
@@ -32,4 +32,4 @@ mkdir -p /tmp/kernel
 mv usr/src/kernels/*/* /tmp/kernel
 
 # exit value
-echo /tmp/kernel
\ No newline at end of file
+export KERNELDIR=/tmp/kernel
```

Comment:
All kernel scripts will now export KERNELDIR at the end.
This way, they are also free to echo whatever they wish to stdout since stdout is no more used.